tests/cypress/integration/Home/commerceHomePage.cy.js

```diff
@@ -127,15 +128,35 @@ describe( 'Commerce Home Page- Live Mode', () => {
 } );
 
 describe( 'Commerce Home Page- Next Steps', () => {
+	const step_id = [
```

Comment:
Once [PR](https://github.com/newfold-labs/wp-module-ecommerce/pull/284) goes live , it will be adding few more items in `next steps` section which we need to update here as well.

Reply:
Okay, noted.

Reply:
This PR does not have any tests breaking, as far as I know these steps won't be shown until we initiate and complete migration via test steps which we are not doing since the process is through a third party.

Once this PR goes live, if possible, we can make these steps appear by setting any value and then verify it.

Reply:
@sangeetha-nayak 
If you look at my earlier comment carefully I never said current tests will break.  What I asked is to add some more additional tests which will be needed after migration PR goes live. 

Just so that we are all on same page I would like to elaborate more on my comment,

- I am aware that migration is handled by 3rd party as discussed by @diwanshuster in standup. 
- But once someone initiates migration & completes it, steps related to migration should appear in `next steps` section. 
- This `next steps` section is a part of Ecommerce module. 
- So our cypress tests need to verify if those steps were added as expected in above scenario. 
- Without those our cypress test coverage will be incomplete. 

So do the needful.

Reply:
New tests will be added around that once the PR goes live. 

The scope of this story [PRESS0-1256](https://jira.newfold.com/browse/PRESS0-1256) is limited to removing text assertions from existing tests and not to add new tests to a feature which is not live yet.

Reply:
- As far as I understand, since migration is a `bigger` change, which also involves 3rd party doing major code changes. 
- It will be good to add `cypress tests` that do a simple verification of `next steps` section and ensure successful E2E completion of migration 
- Pushing this small change along with migration PR itself doesn't seem like a big task and is safe bet according to me.
- We could very well do changes in a separate ticket & PR keeping in mind scope of work for this [JIRA](https://jira.newfold.com/browse/PRESS0-1256)

@diwanshuster you may take the final call on the priority of task to add cypress tests.

Reply:
We can have the initial release without integrating cypress tests mainly relying on manual testing. Having said that, once this is released for 0% users (which will be the case for the initial plugin release behind capability), the next release should have complete test coverage for all next steps options/ecommerce changes pertaining to migration.